### ISSUE TITLE
after latest update encoding doesnt start right away

### ISSUE THREAD

**Issue by loso20p** [WWBN/AVideo-Encoder#62]
ever since i updated encoding doesnt start right away like before. after the file download it would go straight to encoding the file now it takes at least a minute or 2 to start. did yall change something that makes it take longer to get to encoding?

**Comment by DanielnetoDotCom** [comment]
Yes, now, before start to encode, the Encoder extract the images, JPG and GIF, and send them to the streamer.
Then you do not have an static image anymore while is encoding.

**Comment by gamersalpha** [comment]
Hello

Well for me thé image ans the GIF is perfect but hé vidéos don't want Stuart

If i upluoad a vidéo not working

If i import from YouTube works

**Comment by DanielnetoDotCom** [comment]
Hi, I think your text is lang mixed, I dont understand

**Comment by chrissyshiru** [comment]
My video has been encoding for two days. status: encoding. how can i solve this?

**Comment by DanielnetoDotCom** [comment]
If you are encoder admin just delete it from queue.
